app/ModelSerializers/Summit/Presentation/PresentationSerializer.php

```diff
@@ -225,6 +225,8 @@ public function serialize($expand = null, array $fields = array(), array $relati
                                 [],
                                 $params
                             );
+                            $serialized_speaker['presentation_assignment_order'] = $s->getPresentationAssignmentOrder($presentation);
```

Comment:
please use another param name like 'order' this should be transparent to end user

Reply:
change by 'order'